### PR TITLE
[ADMIN] publication of MRA whitepaper

### PR DESCRIPTION
**Pull request by jpradocueva**
The latest changes was an update to the README file to add a link to the PDF and blog on the OMP website:

**Blog**: https://open-manufacturing.org/blog/2021/05/23/manufacturing-reference-architecture/
**PDF**: https://open-manufacturing.org/wp-content/uploads/sites/101/2021/05/OMP_Reference_Architecture_Whitepaper-17-May-21.pdf

### CODE REVIEW

**Comment by jpradocueva** [comment]
License file update to "Patent-Policy-W3C-Mode" as described in the WG Charter.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,49 @@
...
+### A white paper authored by the Manufacturing Reference Architecture Working Group
+
+The Manufacturing Reference Architecture Working Group introduces their view on manufacturing characteristics,
+challenges and use case considerations for architecture design. In addition, the whitepaper discusses required platform capabilities and architecture design principles for a successful implementation. This paper is the initial publication future OMP work will reference an be based upon. 
+
+* [GitHub Version](./Whitepaper/01_Introduction_to_the_OMP_Manufacturing_Reference_Architecture/00_Acknowledgements_and_TOC.md)
+* [Download from Website](https://open-manufacturing.org/wp-content/uploads/sites/101/2021/05/OMP_Reference_Architecture_Whitepaper-17-May-21.pdf)
+* [Blog](https://open-manufacturing.org/blog/2021/05/23/manufacturing-reference-architecture/)
+
+**TODO - Adjust Whitepaper Download URL**
```
Comment (wrichter) [inline README.md]: We should remove the TODO markers now that the URL has been adapted

**Review by wrichter — CHANGES_REQUESTED** [review]
Minor change requested: remove the TODO marker in README.md

File: README.md (hunk 1 above)
Comment (jpradocueva) [reply README.md]: Ups! I will take care of it. Thanks for pointing it out. I thought that I did but clearly I didn't.

**Comment by jpradocueva** [comment]
We need to merge this PR into production.
Today, Tuesday, May 25th at 10:00 am PDT, HubSpot will start an email campaign announcing the publication of the MRA whitepaper. The email will be sent to over 380 users in OMP HubSpot database.

**Pull request merged**